src/irmin-pack/s.ml

```diff
@@ -55,7 +55,44 @@ module type Specifics = sig
   (** [flush t] flush read-write pack on disk. Raises [RO_Not_Allowed] if called
       by a readonly instance.*)
 
-  val gc : ?unlink:bool -> repo -> commit_key -> unit
+  val start_gc :
+    ?unlink:bool ->
+    ?auto_finalise:bool ->
+    throttle:[ `Block | `Skip ] ->
+    repo ->
+    commit_key ->
+    bool Lwt.t
+  (** [start_gc] tries to start the gc process and returns true if the gc is
+      launched.
+
+      If [unlink] is false then temporary files on disk are kept after the gc
+      finished. This option is useful for debugging. The default is true.
```

Comment:
`If [unlink] is false then temporary files and files from the previous generation will be kept on disk after the gc finished. This option is useful for debugging. The default is true.`